test/unit/helpers/xhr/MFA_ENROLL_ACTIVATE_webauthn.js

```diff
@@ -0,0 +1,68 @@
+define({
+  "status": 200,
+  "responseType": "json",
+  "response": {
+    "stateToken": "testStateToken",
+    "expiresAt": "2018-08-17T02:38:18.000Z",
+    "status": "MFA_ENROLL_ACTIVATE",
+    "_embedded": {
+      "user": {
+        "id": "00ukjpMDb9O7CjJbx0g3",
+        "passwordChanged": "2018-08-15T18:31:57.000Z",
+        "profile": {
+          "login": "exampleUser@example.com",
+          "firstName": "Test",
+          "lastName": "User",
+          "locale": "en_US",
+          "timeZone": "America/Los_Angeles"
+        }
+      },
+      "factor": {
+        "id":"fuf52dhWPdJAbqiUU0g4",
+        "factorType":"webauthn",
+        "provider":"FIDO",
+        "vendorName":"FIDO",
+        "_embedded":{
+          "activation":{
+            "rp": {
+              "name": "acme"
+            },
+            "user": {
+              "id": "00u1212qZXXap6Cts0g4",
```

Comment:
might want to change to test data

Reply:
This id value is generated from my local test environment.